ibis/tests/expr/test_table.py

```diff
@@ -1414,6 +1414,20 @@ def test_union(
         setops_table_foo.union(setops_table_baz)
 
 
+def test_union_large():
+    num_cols = 100
+    num_tables = 500
+    num_rows = 100
+    table = ibis.memtable({f"col_{i}": [-1] * num_rows for i in range(num_cols)})
```

Comment:
You can construct this with `ibis.table(schema={...})` instead, which doesn't require a backend.